app/assets/stylesheets/application.scss

```diff
@@ -10,4 +10,26 @@
 
 .remove {
   display: none;
+}
+
+.input_checkbox input {
+  width: auto !important;
+}
+
+.img-show {
+  width: 300px;
```

Comment:
We'll cover this better in your front-end training but try to avoid using `px` or other static values in your CSS as the site won't be responsive :)